### DISCUSSION TITLE
Change from SSO to Login with Email

### DISCUSSION THREAD

**Discussion by zettaivanugen** [outline/outline#5045] [Self Hosting]
Hi all!
We are moving to another Google Workspace and we will not be able to use SSO to login. 
Is there any way we can change the SSO to a "Login with email" setup without losing the accounts and their data? We will not be losing access to the original email accounts but we will be losing access to the "Login with Google".

Thank you!

**Comment by tommoor** [comment]
If you're using getoutline.com then contact hello@getoutline.com and support can help with this transition.

For self-hosted you'll need to go into the database and set enabled=false on the authentication provider.

**Reply by zettaivanugen** [reply]
Thanks @tommoor. 
This prevents us from "Signing in with Google" but doesn't allow us to "Continue with Email". 

When trying to "Continue with Email" and typing the email we were using previously, it redirects us to Google instead of sending us the magic link.

**Reply by tommoor** [reply]
This sounds like a bug then, as you shouldn't be forwarded to a disabled auth provider. Please file and we'll try and get it fixed before the next community edition release.

**Reply by zettaivanugen** [reply]
Thanks. Is there any way we can export all collections including the
private ones? The domain transfer will be happening soon and we don’t want
to lose the data on those accounts. 

The full export only seemed to compile the public collections.

**Reply by oakbrad** [reply]
@zettaivanugen Export function has access to whatever the user/API key has access to.

I'm also interested in removing Google SSO from my self hosted setup so following thread. I occasionally have users who tell me SSO doesn't work and they have to use magic link (had one this morning onboarding someone new) but I haven't been able to track down why this happens to some and not others. None of my users are in my Google workspace so might as well just move everyone to magic link.

**Comment by karan** [comment]
Is there a working path forward for this @oakbrad @zettaivanugen? I am also interested in moving away from my Google oauth profile to a normal login-with-email and not sure if you have tried this.
